IntentsExtension/Tools/IntentsConstants.swift

```diff
@@ -10,4 +10,7 @@ import Foundation
 
 struct IntentsConstants {
     static let noteIdentifierKey = "OpenNoteIntentHandlerIdentifierKey"
+    static let extensionGroupName = Bundle.main.sharedGroupDomain
+    static let simperiumAppID = "chalk-bump-f49"
```

Comment:
This should probably come from `SimplenoteConstants.simperiumAppID`

Please make sure that file doesn't get uploaded!

Reply:
done